migrations/versions/118aa16f565b_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 118aa16f565b
+Revises: cf135a57332e
+Create Date: 2019-02-05 20:16:52.182780
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '118aa16f565b'
+down_revision = 'cf135a57332e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('tablemetadata_queries_link',
+        sa.Column('table_id', sa.Integer(), nullable=False),
+        sa.Column('query_id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['query_id'], ['queries.id'], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(['table_id'], ['table_metadata.id'], ondelete="CASCADE"),
+        sa.PrimaryKeyConstraint('table_id', 'query_id')
+    )
+    op.drop_column(u'table_metadata', 'sample_query')
```

Comment:
Should this automatically migrate the existing sample_queries to the new table?

Reply:
Actually, when I first created the "sample_query" column, it didn't have any UI associated with it and it wasn't used for anything. So it's being dropped here since it's replaced with this entire PR basically.

Reply:
Done

Reply:
Ah, d'oh, thanks for elaborating!